judgments/views/detail.py

```diff
@@ -68,6 +68,13 @@ def get_best_pdf(request, judgment_uri):
 
 
 def detail(request, judgment_uri):
+    context = {}
+
+    if judgment_uri.endswith("press-summary/1"):
```

Comment:
probably for another card. but just reading the yellow boxes. 
We could definitely think about reducing the box link text. 
perhaps. ‘ Download PDF of this judgment’ might work better as the key word is PDF and it’s a bit earlier in the read. 
Also in the body copy it says ‘printing and downloading’ I think the order should swooped because you would have to download before printing physically.

Reply:
> This should probably be `.contains("/press-summary/")`, since we're open to the possibility of multiple press summaries (or other documents of a type). There might be a need for a "get doc type from URL" util function.

@dragon-dxw  Yeah this was something I wanted to ask about.... I was about it unsure as to whether we had decided we'd keep the /1. I thought Rose had said we are certain we would only have one per judgment. Will bring up as a discussion in slack.

Reply:
> probably for another card. but just reading the yellow boxes. We could definitely think about reducing the box link text. perhaps. ‘ Download PDF of this judgment’ might work better as the key word is PDF and it’s a bit earlier in the read. And might fit on one line too. same for XML. Also in the body copy it says ‘printing and downloading’ I think the order should be swopped because you would have to download before printing physically.

@Terry-Price  Yeah definitely worth thinking about and making another ticket if the copy is agreed on 👍